node/full.go

```diff
@@ -396,10 +396,9 @@ func (n *FullNode) Run(ctx context.Context) error {
 		return fmt.Errorf("error while starting data sync service: %w", err)
 	}
 
-	go n.reaper.Start(ctx)
-
 	if n.nodeConfig.Node.Aggregator {
 		n.Logger.Info("working in aggregator mode", "block time", n.nodeConfig.Node.BlockTime)
+		go n.reaper.Start(ctx)
```

Comment:
@gupadhyaya was there a reason to run reaper in non aggregation mode?

Reply:
in based there isnt a need to sign the header right? so we would want to run the reaper but not the signer right?

Reply:
yes, that is correct. no need to sign.

Reply:
added a todo and made an issues